app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -1,5 +1,282 @@
-// This component relies on JavaScript from GOV.UK Frontend
-// = require govuk/components/accordion/accordion.js
+/* global nodeListForEach */
+//  = require ../vendor/polyfills/closest.js
+//  = require ../vendor/polyfills/indexOf.js
+//  = require ../vendor/polyfills/common.js
+
 window.GOVUK = window.GOVUK || {}
-window.GOVUK.Modules = window.GOVUK.Modules || {}
-window.GOVUK.Modules.Accordion = window.GOVUKFrontend
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function GemCAccordion () { }
```

Comment:
This name seems a bit clunky - we only use the `gem-c` prefix for CSS classes. I think `GemAccordion` is fine - I assume `Accordion` is already taken?

Reply:
Ah - makes sense, `GemAccordion` used instead.